php/php-8.2.0-slim/wl-env-repo.sh

```diff
@@ -9,4 +9,5 @@ fi
 
 export WASMLABS_REPO=https://github.com/php/php-src.git
 export WASMLABS_REPO_NAME=php
+export WASMLABS_REPO_BRANCH=php-8.2.0
 
```

Comment:
Now, these `wl-env-repo.sh` files are needed for each language-runtime-flavor.
So php-8.2.0 and php-8.2.0-slim have their own.